### PR TITLE
Added new matched `User-Agent`'s, `Highlighted Stories` and some `UserResponse` properties

### PR DESCRIPTION
**Pull request by bariscck**
New `User-Agent`'s fixed `Google Play` header and LoginWebView related issues like  #173 
Added some missing properties to `UserResponse` like follow counts, profile context vs..
Added `Highlighted Stories` request

### CODE REVIEW

**Comment by bariscck** [comment]
Added `Friendship` response for `follow`, `unfollow` requests instead of `Status` response.

File: SwiftyInsta/API/Client/APIHandler.swift [hunk 1]
```diff
@@ -48,7 +48,7 @@ public class APIHandler {
         /// The queue used to deliver responses. Defaults to `DispatchQueue.global(qos: .utility)`.
         public var queues: Queues
         /// The device. Defaults to a random device.
-        public var device: Device { didSet { headers[Headers.userAgentKey] = device.userAgent }}
+        public var device: Device
```
Comment (sbertix) [inline SwiftyInsta/API/Client/APIHandler.swift:51]: I think you forgot to update the head fields again.

Comment (bariscck) [reply SwiftyInsta/API/Client/APIHandler.swift:51]: No, I don't forgot. If setting `User-Agent` in this way `LoginWebView` returns `Login failed` error.

Comment (bariscck) [reply SwiftyInsta/API/Client/APIHandler.swift:51]: Counters on `User`is part of `handler.users.user` request; If send request with `primaryKey` counter values returns. I'm using it in my app.

Comment (bariscck) [reply SwiftyInsta/API/Client/APIHandler.swift:51]: I tested new `User-Agents` in different devices, networks and I've also tested with your repo `Swiftagram` it works for any request.

Comment (bariscck) [reply SwiftyInsta/API/Client/APIHandler.swift:51]: I didn't tested without `LoginWebView`.

**Comment by bariscck** [comment]
Counters on Useris part of handler.users.user request; If send request with primaryKey counter values returns. I'm using it in my app.

I tested new User-Agents in different devices, networks and I've also tested with your repo Swiftagram it works for any request with `LoginWebView`.

I didn't tested without LoginWebView.

**Comment by sbertix** [comment]
Replying here so it's easier to follow. 

I'm perfectly aware `User`-like instances when called directly return counters (for instance in /users/:id/info), but they don't in most other cases when `User` like structures are returned: followers, media users, tags, etc. I was just stating that and that's reason why they were never included, cause they are optional fields, definitely not required in the endpoint response schema.
But I finally agree with adding them, definitely not a problem 💪

About the user agent, there's no point in even storing it then since you write it in `Device` but never read it, as requests fallback to the default one.\
So I suggest removing it entirely then.

**Comment by bariscck** [comment]
Just the better understand; should I remove the parts related to `device` or `User-Agent`?

**Comment by sbertix** [comment]
Just the definition of `userAgent` from `Device`: it's never read so it serves no purpose.

**Comment by bariscck** [comment]
> Just the definition of `userAgent` from `Device`: it's never read now, so it serves no purpose. 😊

😊 Its removed.

**Comment by sbertix** [comment]
Great 💪 
I'll deal with linting (there are some trailing whitespaces, so it might fail in Travis, but no worries), merging this and bumping the pod version, np.

Do you know why the CI report is not showing up here? 😢  @TheM4hd1

**Comment by TheM4hd1** [comment]
@sbertix no idea what happened, it shows on travis panel, but not sure why it disabled here.

**Pull request merged**